src/crab/cfg.hpp

```diff
@@ -619,8 +619,13 @@ std::vector<std::string> stats_headers();
 
 std::map<std::string, int> collect_stats(const cfg_t&);
 
-cfg_t prepare_cfg(const InstructionSeq& prog, const program_info& info, bool simplify, bool check_for_termination,
-                  bool must_have_exit = true);
+struct prepare_cfg_options {
+    bool simplify = true;
+    bool check_for_termination = true;
+    bool must_have_exit = true;
+};
+
+cfg_t prepare_cfg(const InstructionSeq& prog, const program_info& info, const prepare_cfg_options& options);
```

Comment:
I don't remember - is it not possible to set default `options=prepare_cfg_options{}`?